sparse_ho/grad_search_CV.py

```diff
@@ -10,7 +10,8 @@ def grad_search_CV(
         verbose=True, cv=5, random_state=0, test_size=0.33,
         tolerance_decrease='constant', tol=1e-5,
         t_max=10000, beta_star=None,):
-
+    """Depreciated fucntion, CV is now a proper criterion
```

Comment:
```suggestion
    """Deprecated function, CV is now a proper criterion
```
if it's deprecated should you just remove it?